src/components/language-switcher.js

```diff
@@ -23,24 +22,30 @@ function LanguageSwitcher({ slug }) {
 
   return (
     <div className="language-switcher">
-      <Label htmlFor="input-dropdown">Translate</Label>
-      <Dropdown
-        className="language-switcher__dropdown"
-        id="input-dropdown"
-        name="input-dropdown"
-        defaultValue={currentLang}
-        onChange={handleChange}
-      >
-        {languages.map((lng) => {
-          return (
-            <option name="lng" value={lng.code} key={lng.code}>
-              {lng.label}
-            </option>
-          );
-        })}
-      </Dropdown>
-      {/* <Button type="submit">Translate</Button> */}
-      {/* TODO: good to include a submit button (for accessibility reasons)*/}
+      <Grid row gap>
```

Comment:
This is still in progress - trying to use grid instead of flex to position the dropdown

Reply:
This component will end up being in the banner - so disregard any styling changes here for now